low-level/files/opt/seagate/sspl/prerequisites/sspl-prereqs.sh

```diff
@@ -0,0 +1,320 @@
+            CNTRLR_A="$2"
+            shift 2 ;;
+        --Ap|--cntrlr_a_port)
+            [ -z "$2" ] && echo "ERROR: Controller A Port not provided" && exit 1;
+            CNTRLR_A_PORT="$2"
+            shift 2 ;;
+        -B|--cntrlr_b)
+            [ -z "$2" ] && echo "ERROR: Controller B IP not provided" && exit 1;
+            CNTRLR_B="$2"
+            shift 2 ;;
+        --Bp|--cntrlr_b_port)
+            [ -z "$2" ] && echo "ERROR: Controller B Port not provided" && exit 1;
+            CNTRLR_B_PORT="$2"
+            shift 2 ;;
+        -U|--cntrlr_user)
+            [ -z "$2" ] && echo "ERROR: Controller user name not provided" && exit 1;
+            CNTRLR_USER="$2"
+            shift 2 ;;
+        -P|--cntrlr_pass)
+            [ -z "$2" ] && echo "ERROR: Controller password not provided" && exit 1;
+            CNTRLR_PASS="$2"
+            shift 2 ;;
+        -T|--taregt_build)
+            [ -z "$2" ] && echo "ERROR: Target build not provided" && exit 1;
+            TARGET_BUILD="$2"
+            shift 2 ;;
+        --i|--bmc_ip)
+            [ -z "$2" ] && echo "ERROR: BMC IP not provided" && exit 1;
+            BMC_IP="$2"
+            shift 2 ;;
+        --u|--bmc_user)
+            [ -z "$2" ] && echo "ERROR: BMC user not provided" && exit 1;
+            BMC_USER="$2"
+            shift 2 ;;
+        --p|--bmc_pass)
+            [ -z "$2" ] && echo "ERROR: BMC password not provided" && exit 1;
+            BMC_PASS="$2"
+            shift 2 ;;
+        *)
+            echo "ERROR: Unknown option provided: $1"
+            exit 1 ;;
+        esac
+    done
+}
+
+parse_args "$@"
+
+# Cleanup
+cleanup(){
+    systemctl stop rabbitmq-server
+    yum remove -y consul rabbitmq-server cortx-utils
+    yum remove -y cortx-sspl.noarch
+}
+[ "$do_cleanup" == "true" ] && cleanup
+
+# Setup common & 3rd_party repos
+echo "Setup repos" 2>&1 | tee -a ${LOG_FILE}
+CORTX_MONITOR_BASE_URL="https://raw.githubusercontent.com/mariyappanp/cortx-monitor/EOS-15396_self_prv"
+curl $CORTX_MONITOR_BASE_URL/low-level/files/opt/seagate/sspl/prerequisites/setup_repos.sh -o setup_repos.sh
+chmod a+x setup_repos.sh
+
+if [ "$disable_sub_mgr" == "true" ] && [ -n "$TARGET_BUILD" ]; then
+    ./setup_repos.sh --disable-sub-mgr -t $TARGET_BUILD 2>&1 | tee -a ${LOG_FILE}
+elif [ "$disable_sub_mgr" == "true" ]; then
+    ./setup_repos.sh --disable-sub-mgr 2>&1 | tee -a ${LOG_FILE}
+elif [ -n "$TARGET_BUILD" ]; then
+    ./setup_repos.sh -t $TARGET_BUILD 2>&1 | tee -a ${LOG_FILE}
+else
+    ./setup_repos.sh 2>&1 | tee -a ${LOG_FILE}
+fi
+rm -rf setup_repos.sh
+echo -e "\nSetup repos - Done" 2>&1 | tee -a ${LOG_FILE}
+
+
+# Install prereq script dependencies
+yum install -y python3
+pip3 install configobj
+
+# Install required RPMS if not available
+"$install_3rd_party_packages" && {
+    rpm -qa | grep "cortx-py-utils" || {
+        yum install -y cortx-py-utils 2>&1 | tee -a ${LOG_FILE}
```

Comment:
please log in CAPITAL for each such installation

Reply:
Okay